package/src/skia/__tests__/ColorFilter.spec.ts

```diff
@@ -0,0 +1,66 @@
+import fs from "fs";
+import path from "path";
+
+import type { SkColorFilter } from "../types/ColorFilter/ColorFilter";
+
+import { setupSkia, processResult } from "./setup";
+
+describe("Color Filters", () => {
+  it("Color Matrix 1", () => {
```

Comment:
Should we follow the jest naming convention in test names?

`it("should apply the colour matrix to the output  when .....")`

It often makes test clearer to read and understand since it forces us to define both the expected outcome and the conditions given.